.github/workflows/changelog_verifier.yml

```diff
@@ -0,0 +1,18 @@
+name: "Changelog Verifier"
+on:
+  pull_request:
+    types: [opened, edited, review_requested, synchronize, reopened, ready_for_review, labeled, unlabeled]
+
+jobs:
+  # Enforces the update of a changelog file on every pull request
+  verify-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - uses: dangoslen/changelog-enforcer@v3
+        with:
+          skipLabels: "autocut, skip-changelog"
```

Comment:
why autocut is included? We can add "skip-changelog" to autocuts if needed.

Reply:
Followed what OpenSearch core repo is doing. I think most of commit done by auto cut won't get included in CHANGELOG. Yes. Adding `skip-changelog` in the autocut might be better if some of the autocut need to get included in the CHANGELOG. Do we have such case do you know?

Reply:
I don't have any particular reference, but I can image it's could be a fix for a broken distribution build, or other type of infrastructure change, like build/publish script

Reply:
I'm fine with having auto-cut if that what core is doing, we can revert it later if that doesn't work for us. Just want to understand the reason behind adding it

Reply:
Those won't be auto cut PR I believe. Issue could be auto cut but not PR.
The reason that we don't include `autocut` here is because I don't think autocut can add new line in CHANGELOG file intelligently. That mean, it won't be auto but semi-auto which need manual update on the PR.